AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.ts

```diff
@@ -42,9 +41,6 @@ export class AddParticipantComponent extends AddParticipantBaseDirective impleme
     showConfirmationRemoveParticipant = false;
     removerFullName: string;
     localEditMode = false;
-    isExistingHearing: boolean;
```

Comment:
why remove these fields?

Reply:
They're unused anywhere in the component

Reply:
good find!